dist/index.mjs

```diff
@@ -34,7 +39,7 @@ class Applications {
             endpoint: `/applications/${applicationId}`,
         });
     }
-    async label(applicationId, data, options) {
+    async label(applicationId, data, isCatalog, options) {
```

Comment:
the id for a catalog application can be a user friendly string, but user should not have to always pass a boolean here every time they want to label and also pass options

catalog application:
`applications.label("sentiment", data, options)`

user application:
 `applications.label("some-uuid-here", data, options)`
 
 ```typescript
 const endpoint = isUuid(applicationId) ? `/applications/${applicationId}/label?${params.toString()}` : `/catalog/${applicationId}/predict?${params.toString()}`;
 ```